ksql-functional-tests/src/test/resources/rest-query-validation-tests/pull-queries-against-materialized-aggregates.json

```
@@ -955,6 +955,25 @@
         "message": "Column 'AGGREGATE.WINDOWSTART' cannot be resolved",
         "status": 400
       }
+    },
+    {
+      "name": "window bounds in projection UDF",
+      "statements": [
+        "CREATE STREAM INPUT (IGNORED INT) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE TABLE AGGREGATE AS SELECT COUNT(1) AS COUNT FROM INPUT WINDOW TUMBLING(SIZE 1 SECOND) GROUP BY ROWKEY;",
+        "SELECT ROWKEY, TIMESTAMPTOSTRING(WINDOWSTART, 'yyyy-MM-dd HH:mm:ss Z') AS WSTART, TIMESTAMPTOSTRING(WINDOWEND, 'yyyy-MM-dd HH:mm:ss Z') AS WEND, COUNT FROM AGGREGATE WHERE ROWKEY='10';"
```

please make sure to specify `'UTC'` as the time zone :) it passes on your local machine, it passes on jenkins, but it breaks on our plebeian PTC machines.
```suggestion
        "SELECT ROWKEY, TIMESTAMPTOSTRING(WINDOWSTART, 'yyyy-MM-dd HH:mm:ss Z', 'UTC') AS WSTART, TIMESTAMPTOSTRING(WINDOWEND, 'yyyy-MM-dd HH:mm:ss Z', 'UTC') AS WEND, COUNT FROM AGGREGATE WHERE ROWKEY='10';"
```

---

Good to know - will pick up in next PR.